frontend-react/src/routes.js

```diff
@@ -70,6 +71,11 @@ const routes = [
     key: routeKeys.NOTFOUND,
     component: lazy(() => import('./views/NotFound'))
   },
+  {
+    path: '/drive',
+    key: routeKeys.DRIVE,
+    component: lazy(() => import('./views/AddDrive'))
```

Comment:
Sorry for pointing it out only now, but there's one inconsistency - this view is mapped under /drive, key is named DRIVE and the class is named DriveView, but the directory is named AddDrive. Can you rename it again to Drive?

I'm sorry for pointing it out this late but I just noticed that.

Reply:
No problem, I also was thinking about this inconsistency. Thank you for pointing this.